packages/orbit-components/cypress/integration/pages/text-styles.test.ts

```diff
@@ -0,0 +1,197 @@
+import { defaultFoundation } from "@kiwicom/orbit-design-tokens";
+import color from "onecolor";
```

Comment:
I think better will be add here Cypress screenshot test instead of actually adding a new dependency, or new tokens (that are supposed to be used only in tests from what I see), it looks too comprehensive imo :) 

So I suggest to: 
- remove onecolor dep
- remove those transparency tokens
- remove those color tests
- use screenshot testing instead

Reply:
Tokens are not used just for tests, as mentioned above. I am not worried about using `onecolor` for color assertions. It's a devDep just for testing, so I honestly see no harm in here… and it seems faster and equally (or even more) reliable than screenshot testing, no?
By using screenshot you suggest using loki? Or just screenshot testing in cypress?

Reply:
Just screenshot with cypress. Cypress is already running on CI, Loki has to be run manually.

Reply:
Cypress [does not allow visual testing natively](https://docs.cypress.io/guides/tooling/visual-testing). It has some open-source plugins for that, but I can believe those extra dependencies are even bigger than just this one. I do agree we need visual regression testing asap, as even [Cypress mentions that asserting CSS is not ideal](https://docs.cypress.io/guides/tooling/visual-testing#:~:text=You%20could%20technically%20write%20a%20functional%20test%20asserting%20the%20CSS%20properties%20using%20the%20have.css%20assertion%2C%20but%20these%20may%20quickly%20become%20cumbersome%20to%20write%20and%20maintain%2C%20especially%20when%20visual%20styles%20rely%20on%20a%20lot%20of%20CSS%20styles.).

My suggestion is that, as we already have this for Box, Stack and now Text, we don't block this PR for this and we actually think of a testing solution to implement. Let me know what you think.

Reply:
I would test only functionality and media queries, testing of colors then I would completely skip. We might consider to try out some of the plugins, as it seems to me as better alternative than using Loki.

Reply:
It is not about testing colors _per se_. It is about testing the effects of the `withBackground` and `type` props, that I feel should not be left untested as they're part of the core of the component